protocols/randhound/proof_test.go

```diff
@@ -0,0 +1,190 @@
+package randhound_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/dedis/cothority/protocols/randhound"
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/edwards"
+	"github.com/dedis/crypto/random"
+)
+
+func TestProof(t *testing.T) {
+
+	suite := edwards.NewAES128SHA256Ed25519(false)
+
+	// 1st set of base points
+	g1, _ := suite.Point().Pick([]byte("G1"), random.Stream)
+	h1, _ := suite.Point().Pick([]byte("H1"), random.Stream)
+
+	// 1st secret value
+	x := suite.Scalar().Pick(random.Stream)
+
+	// 2nd set of base points
+	g2, _ := suite.Point().Pick([]byte("G2"), random.Stream)
+	h2, _ := suite.Point().Pick([]byte("H2"), random.Stream)
+
+	// 2nd secret value
+	y := suite.Scalar().Pick(random.Stream)
+
+	// Create proofs
+	g := []abstract.Point{g1, g2}
+	h := []abstract.Point{h1, h2}
+	p, err := randhound.NewProof(suite, g, h, nil)
+	if err != nil {
```

Comment:
Possibility of using
`log.ErrFatal(err)`
which `fatal`s if `err != nil`
or use
`require.Nil(t, err)`